src/components/inquiry_artwork_brick/index.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import Icon from "../icon"
+import ArtworkMetadata from "./metadata"
+
+const InquiryArtworkBrick = styled.div`
+  position: relative;
+`
+
+const ImageContainer = styled.div`
+  height: 350px; 
```

Comment:
consider adding variable sizes based on the size of the screen (e.g. desktop vs mobile). We can do that in a separate PR when we add mobile support though.

For instance:

```
  height: ${props => props.size == Sizes.small ? 150 : 350 }
  ...
```